engines/dreamweb/newplace.cpp

```diff
@@ -126,7 +126,7 @@ void DreamWebEngine::lookAtPlace() {
 	findNextColon(&string);
 
 	if (_ttsMan != nullptr && ConfMan.getBool("tts_enabled_objects"))
-		_ttsMan->say((const char *)string);
+		_ttsMan->say((const char *)string, Common::TextToSpeechManager::DROP, _textEncoding);
```

Comment:
Is there a reason to change the behaviour here (and in the other places where you made the change) from `INTERRUPT` (the default) to `DROP`?

If not you can simply write `_ttsMan->say((const char *)string, _textEncoding);`

Reply:
I thought misunderstood the default as `DROP`, and resolved to add it to all the places I intended with `_ttsMan->say((const char *)string, _textEncoding);`. It seemed that I need 3 inputs for that.

Reply:
I was wrong! They're all good now